Tipster/Tipster.cpp

```diff
@@ -104,8 +136,31 @@ Tipster::MouseDown(BPoint pt)
 void
 Tipster::UpdateTip()
```

Comment:
Currently tipster-tips.txt needs to start with a "%", else there's some mess-up that'll end up showing the URL of the first tip. Since the "fortunes" format doesn't have a "%" at the start, you should try to not need it for Tipster. We may load other fortunes-style files in the future.